bin/gobuild.sh

```diff
@@ -59,5 +50,5 @@ while read line; do
 done < "${BUILDINFO}"
 
 # forgoing -i (incremental build) because it will be deprecated by tool chain. 
-time GOOS=${GOOS} GOARCH=${GOARCH} ${GOBINARY} build ${V} ${GOBUILDFLAGS} -o ${OUT} \
+time CGO_ENABLED=0 ${GOBINARY} build ${V} ${GOBUILDFLAGS} -o ${OUT} \
```

Comment:
It looks like another case where non-MAC builds will be affected. Please restrict the change to mac builds, and open separate PRs for changing how we build on linux.